momentum_scanner_intraday.py

```diff
@@ -57,6 +62,39 @@ def get_risk_query(token, stoploss=0.05, profittaking=0.05):
     print("enter at:", price, "stop-loss:", slprice, "profit-taking:", ptprice)
     return slprice, ptprice
 
+def find_chain(dex):
+    """
+    find the chain on which a dex is deployed
+    """
+    chain_cg = None
+    chain_cgterminal = None
+    for i in dex_chain:
+        if i['id'] == dex:
+            chain = i['chain']
+            chain_cg = network_map_cg[chain]
+            chain_cgterminal = network_map_cgterminal[chain]
+    return chain_cg, chain_cgterminal
+
+def get_cgterminal_url(chain, contract_addr):
+    return 'https://api.geckoterminal.com/api/v2/networks/'+chain+'/tokens/'+contract_addr+'/pools'
+
+def find_best_reserve(url):
+    """
+    find best reserve via cg-terminal api
+    """
+    re = requests.get(url).json()
```

Comment:
Change the name from `re` to something that names what this json contains.

Reply:
Done